pensieve/pensieve.py

```diff
@@ -353,30 +347,22 @@ def gen_mem_dict(self, character, verb_cut=500, name_cut=100):
         mem_people = []
         if isinstance(character, str):
             character = [character]
-        for name in self.words['names']:
-            if name not in character:  # Ignore redundant mention of subject
+        for name in self.extract_names():
+            if name not in character:
                 mem_people.append(name)
         if len(mem_people) == 0:
-            mem_people.append('alone')  # Some scenes have no other names
-        mem_places, mem_things = self.sanitize_places_and_things(name_cut)
-        mem_activities = self.gen_mem_activity(verb_cut)
+            mem_people.append('alone')
```

Comment:
This can be a dangerous default....alone isnt a person....its a mood....It is okay to NOT have people in a memory....but maybe put 'alone' in moods? Also by defaulting to this....you are creating a high degree node in the database...people do not always feel alone when they are by themselves....
worth reconsidering this one, or show some data to support it